resources/models/reservation.py

```diff
@@ -24,6 +24,7 @@
     get_dt, save_dt, is_valid_time_slot, humanize_duration, send_respa_mail,
     DEFAULT_LANG, localize_datetime, format_dt_range
 )
+from resources.models.utils import build_reservations_ical_file
```

Comment:
This could be among the other `.utils` imports above.

Reply:
Good catch, done.